app/views/people/_member.html.haml

```diff
@@ -3,6 +3,11 @@
     - if member.respond_to?(:full_name) && member.respond_to?(:graph_id)
       %li
         %a{:href => "/people/#{member.graph_id}"}= member.full_name
+
+        %li= "#{t('current_lord')}" if member.statuses.include?('Current Lord')
```

Comment:
So in parliament-ruby there is a statuses decorator which returns an array of statuses, as a member could have 0, 1 or 2 different statuses.  I have changed this partial now to use that and join the different statuses in the array.